### DISCUSSION TITLE
Unable to utilise private NuGet feed as part of lambda packaging via CLI

### DISCUSSION THREAD

**Discussion by mark-hopper** [aws/aws-extensions-for-dotnet-cli#328] [Q&A] [answered]
I've seen a few discussions on this but no real clear-cut way of doing this. Essentially, I'm looking at creating a native AOT .NET 8 lambda function via the dotnet lambda package command. This works great when you have no private feeds which need to be restored from... the moment you introduce one you get "Unable to load the service index for source".

Normally, if we were doing a docker build to then utilise as a docker image, we would use the Azure Artifacts Credential Provider (as our private feed is in Azure Artifacts). But because this is a docker run command we cannot do that.

The only way I can see of doing this, is to prior to running the package command, add the username/password to the Nuget.Config file against the project/solution, which does work.

**Comment by bhoradc** [comment, marked as answer]
Hello @mark-hopper, 

Thank you for opening this discussion. I see that you have submitted a [Feature request](https://github.com/aws/aws-extensions-for-dotnet-cli/issues/329) for the same. I would want to inform you that the team did acknowledge it as a valid Feature request and will work on it based on prioritization. 

Regards, 
Chaitanya
